lexicons/org/hypercerts/funding/receipt.json

```diff
@@ -0,0 +1,70 @@
+{
+  "lexicon": 1,
+  "id": "org.hypercerts.funding.receipt",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Records a funding receipt for a payment from one user to another user. It may be recorded by the recipient, by the sender, or by a third party. The sender may remain anonymous.",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": [
+          "to",
```

Comment:
@holkexyz Shouldn't `from` be required in funding receipts? Even with anonymous donations, I think it's better to require explicit declaration that they're anonymous rather than not being able to distinguish between that and "oops the code accidentally forgot to include that field".